Microsoft.Toolkit.Uwp.UI/Extensions/DependencyObjectExtensions.cs

```diff
@@ -543,6 +543,44 @@ static IEnumerable<DependencyObject> ThrowArgumentOutOfRangeExceptionForInvalidS
             };
         }
 
+        /// <summary>
+        /// Find all first level descendant elements of the specified element. This method can be chained
+        /// with INQ calls to add additional filters or projections on top of the returned results.
```

Comment:
```suggestion
        /// with LINQ calls to add additional filters or projections on top of the returned results.
```

Reply:
Fixed in 99f9f4e1f998f9c40cb77c6035ca8c1844128c88.